crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -444,7 +444,7 @@ impl<'w, 's, T: Component> SystemParamFetch<'w, 's> for ResMutState<T> {
     }
 }
 
-/// The [`SystemParamState`] of `Option<ResMut<T>>`.
+/// The [`SystemParamState`] of [`Option<ResMut<T>>`].
```

Comment:
This make it link to `Option` in the rust doc... Could you also add a link to `ResMut<T>`?

Reply:
Done

Reply:
thanks!